index.html

```diff
@@ -7,13 +7,109 @@
     <!-- Here we import the Boostrap CSS stylesheet -->
     <link href="https://cdn.jsdelivr.net/npm/bootstrap@5.0.0-beta1/dist/css/bootstrap.min.css" rel="stylesheet" integrity="sha384-giJF6kkoqNQ00vy+HMDP7azOuL0xtbfIcaT9wjKHr8RbDVddVHyTfAAsrekwKmP1" crossorigin="anonymous">
     <!-- Add a link to your custom CSS here -->
-
+    <script src="https://kit.fontawesome.com/d50d8d1a45.js" crossorigin="anonymous"></script>
+    <link rel="stylesheet" href="css/main.css">
     <title>Responsive grid project</title>
 </head>
-
 <body>
+   <header>
+     <div class="container-fluid">
+       <div class="row">
+         <div class="col-6">
+           <h1 class="header-title">
+             <div class="square-device"></div>
+                Maya Nelson
+             <span class="title" style="font-size: 25px; font-weight: bolder;margin-left: 7px;">&#47PROJECT MANAGER</span>
+           </h1>
+           
+         </div>
+         <div class="col-6">
+          <nav class="navbar navbar-expand-md ">
+            <i class="fas fa-align-right"></i>
+             <div class="menu">
+              <a href="#">ABOUT ME</a>
+              <a href="#">RESUME</a>
+              <a href="#">PROJECTS</a>
+              <a href="#">CONTACT</a>  
+             </div>
+          </nav>
+         </div>
+       </div>
+     </div>
+   </header>
+
+
+   <main class="container-fluid">
+       <div class="row sec1">
```

Comment:
Can you use the more semantic `section` tag foe your top level blocks instead of a `div`?